### DISCUSSION TITLE
Layout Option for Tree View structures

### DISCUSSION THREAD

**Discussion by lasseklein** [directus/directus#18651] [Feature Requests]
### Summary

A Layout Option for presenting a Tree View hierarchy on the top level Content view

### Basic Example

The proposed view already exist when entering an item. It would allow the user to see the tree structure directly on the top level, and arrange items visually

<img width="812" alt="Screenshot 2023-05-18 at 20 03 27" src="https://github.com/directus/directus/assets/685735/e3da4d6e-6792-47eb-a1f4-f3a0048b3edb">

### Motivation

The main view for a Tree View collection does not allow for viewing the hierarchy

<img width="409" alt="Screenshot 2023-05-18 at 20 15 47" src="https://github.com/directus/directus/assets/685735/cbc68882-5eb1-4238-8b6b-991e07268035">

### Detailed Design

Add a Layout Option that is only available for Tree View structures to present the hierarchy visually

### Must Have:
- Same presentation as today when editing an item with a tree view, but on the top level in Collections

### Drawbacks

Can't think of any

### Alternatives

Content editors often prefer hierarchical presentation when evaluating CMS platforms. Directus already has the functionality for this, but it is a little hard to find and it is not supported on the top level so editors might miss the functionality.

### Adoption Strategy

Existing items might be presented in the previously selected Layout, and new items could use Tree View Layout as default

**Comment by rijkvanzanten** [comment]
I'm aware it's super light on details at the moment, but how does this differ from https://github.com/directus/directus/discussions/3054 ? 🙂

**Comment by lasseklein** [comment]
I agree that this seems to be the same thing. Sorry I didn’t find this helpful when I searched before

**Comment by rijkvanzanten** [comment]
Closing this as a duplicate of #3054. I'll make sure to reference this over there so the new insights don't get lost 🙂
